src/rez/package_order.py

```diff
@@ -754,6 +754,104 @@ def from_pod(cls, data):
         return cls(packages=data["packages"])
 
 
+class PyPAPackageOrder(PackageOrder):
+    """A package order that allows for package ordering according to PEP440/PyPA.
+
+    A prerelease argument can be specified to order prerelease versions in front
+    of release versions
+
+    For example, given the versions [1.0b2, 1.0a1, 1.0, 1.1, 1.0rc1, 1.1b2],
+    an orderer initialized with ``prerelease=""`` would give the order:
+     [1.1, 1.1b2, 1.0, 1.0rc1, 1.0b2, 1.0a1].
+    an orderer initialized with ``prerelease="b"`` would give the order:
+     [1.1b2, 1.1, 1.0b2, 1.0rc1, 1.0, 1.0a1].
+
+    """
+    name = "pypa"
+
+    def __init__(self, prerelease: Literal["", "rc", "b", "a"] = "", packages=None):
```

Comment:
I think this should be `Optional[Literal["rc", "b", "a"]] = None`.  I prefer `None` to `""` as a "empty" default, but also, there's a place below where we pass `None` to this argument (Someone should get mypy running on this project). 

Also the type of `packages` is ` Optional[Iterable[str]]`

Reply:
:+1: